contrib/sssd.spec.in

```diff
@@ -10,25 +8,10 @@
 
 %define _hardened_build 1
 
-    %global enable_polkit_rules_option --disable-polkit-rules-path
-
 # Determine the location of the LDB modules directory
 %global ldb_modulesdir %(pkg-config --variable=modulesdir ldb)
 %global ldb_version 1.2.0
 
-    %global with_cifs_utils_plugin 1
```

Comment:
Current version of sssd.spec.in builds `cifs-idmap-plugin`, `kcm` and `gdm_pam_extensions` only on Fedora and RHEL.
What was the reason?
If the message was, we don't support those components on those platforms, we lose this message.

Reply:
We never supported anything else but Fedora and RHEL in upstream spec file. Conditions like these `%if (0%{?fedora} || 0%{?rhel} > 7)` are there only to provide backward compatibility with older RHEL. The spec file I'm proposing aims only for Fedora 34+ and RHEL-9+ to drop all unnecessary stuff.

Reply:
Ok.